twentytwenty-blocks/functions.php

```diff
@@ -52,29 +52,19 @@ function twentytwentyblocks_theme_support()  {
 			'editor-color-palette',
 			array(
 				array(
-					'name'  => __( 'Primary', 'twentytwenty-blocks' ),
-					'slug'  => 'primary',
-					'color' => '#0073AA',
-				),
-				array(
-					'name'  => __( 'Secondary', 'twentytwenty-blocks' ),
-					'slug'  => 'secondary',
-					'color' => '#005177',
+					'name'  => __( 'Background', 'twentytwenty-blocks' ),
```

Comment:
Getting some unexpected behavior when I try to apply custom colors:

<img width="985" alt="Screen Shot 2020-06-25 at 10 23 10 AM" src="https://user-images.githubusercontent.com/5375500/85738337-f436bb80-b6cd-11ea-8e53-b15bb7645f2a.png">

Reply:
Very weird! 🤔 I'll take another look at that.

Reply:
Would you mind re-testing this? I can't reproduce anymore.

Reply:
Confirmed that this issue is resolved on my end.